website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
I tried this originally, but it will not work with Odoo SA's method of allowing definition of animatable elements in template, because the animations require specific selector. I emulated that method [here](https://github.com/OCA/website/pull/365/files/49144005bd37ab957519aa0e4ccd8fe03728d85d#diff-6d557f05ade49724e2f41b75a4dae43aR30) in order to allow for somewhat cross-compatibility.

There was another issue too, but I can't remember TBH.